commit ebe797f25f68f28581f46a9cb9c1997ac15c39a0
Author: Michael S. Tsirkin <mst@redhat.com>
Date:   Thu Jun 30 15:10:57 2022 -0400

    virtio: VIRTIO_HARDEN_NOTIFICATION is broken
    
    This option doesn't really work and breaks too many drivers.
    Not yet sure what's the right thing to do, for now
    let's make sure randconfig isn't broken by this.
    
    Fixes: c346dae4f3fb ("virtio: disable notification hardening by default")
    Cc: "Jason Wang" <jasowang@redhat.com>
    Signed-off-by: Michael S. Tsirkin <mst@redhat.com>
    Acked-by: Jason Wang <jasowang@redhat.com>

diff --git a/drivers/virtio/Kconfig b/drivers/virtio/Kconfig
index e1556d2a355a..afb9051e0125 100644
--- a/drivers/virtio/Kconfig
+++ b/drivers/virtio/Kconfig
@@ -31,11 +31,12 @@ if VIRTIO_MENU
 
 config VIRTIO_HARDEN_NOTIFICATION
         bool "Harden virtio notification"
+        depends on BROKEN
         help
           Enable this to harden the device notifications and suppress
           those that happen at a time where notifications are illegal.
 
-          Experimental: Note that several drivers still have bugs that
+          Experimental: Note that several drivers still have issues that
           may cause crashes or hangs when correct handling of
           notifications is enforced; depending on the subset of
           drivers and devices you use, this may or may not work.